docs/guide/customizing_form_builder_email_notifications.md

```diff
@@ -0,0 +1,28 @@
+# Customizing Form Builder email notifications
```

Comment:
Please, move to https://doc.ezplatform.com/en/latest/guide/extending_form_builder/#configure-email-notifications